storage/storage.cpp

```diff
@@ -168,6 +168,11 @@ Country const & Storage::CountryByIndex(TIndex const & index) const
 
 void Storage::GetGroupAndCountry(TIndex const & index, string & group, string & country) const
 {
+  // @TODO(bykoianko) This method can work faster and more correctly.
+  // 1. You need parrents (group) - use m_countries "tree".
+  // 2. You need country - it's TIndex now.
+  // 3. You need to use twine instead of FileName2FullName and FullName2GroupAndMap
```

Comment:
Could you please translate this comment to Russian? I can't understand what do you mean here.